libraries/package-extractor/src/PackageExtractor.ts

```diff
@@ -93,6 +93,16 @@ export interface IExtractorProjectConfiguration {
    * The absolute path to the project.
    */
   projectFolder: string;
+  /**
+   * A list of file patterns to include when extracting this project. If undefined,
+   * all files will be included.
+   */
+  patternsToInclude?: string[];
+  /**
+   * A list of file patterns to exclude when extracting this project. If undefined,
+   * no files will be excluded.
+   */
+  patternsToExclude?: string[];
```

Comment:
Mention in the doc comment what kind of pattern it is. Glob? `.gitignore`-style?